Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -503,7 +525,9 @@ def integration_health_check(proxies):
                 response = temp
 
             # get the value in the time field
-            hit_date = str(response.get('hits', {}).get('hits')[0].get('_source').get(str(TIME_FIELD)))
+            source = response.get('hits', {}).get('hits')[0].get('_source', {})
+            hit_date = str(get_value_by_dot_notation(source, str(TIME_FIELD))) if '.' in TIME_FIELD else (
+                str(source.get(str(TIME_FIELD))))
```

Comment:
it's ok, but redundant
```suggestion
            hit_date = str(get_value_by_dot_notation(source, str(TIME_FIELD)))
```